src/Geom_Box.f90

```diff
@@ -107,6 +108,15 @@ MODULE Geom_Box
   !> @copydetails Geom_Box::isequal_OBBoxType
   MODULE PROCEDURE isequal_OBBoxType
 ENDINTERFACE
+
+!> @brief Generic interface for assignment operator (=)
+!>
+!> Adds assignment capability for OBBox types.
+INTERFACE ASSIGNMENT(=)
+  !> @copybrief GeomPoints::assign_OBBoxType
+  !> @copydetails GeomPoints::assign_OBBoxType
+  MODULE PROCEDURE assign_OBBoxType
+ENDINTERFACE
```

Comment:
So... why is this actually necessary?  Fortran has intrinsic assignment for derived types.  Since there are no pointer components or anything like that, it seems like the intrinsic assignment should be sufficient (and more robust since we don't need to ever add components to an assignment procedure we define)

Reply:
Understood.  These types aren't going to change, but your point is valid.  I'll try deleting the assignment overload and seeing if the tests still pass.

Reply:
Tests passed.  Removed.